content/docs/transforms/examples.md

```diff
@@ -0,0 +1,283 @@
+---
+metaTitle: "Example Transforms"
+metaDescription: "Example transform scripts for automating qri dataset updates"
+weight: 3
+---
+
+import InfoBlock from '../../../src/components/InfoBlock.js'
+
+<InfoBlock>
+  This documentation is for qri CLI (command line) only. Composing Transform scripts on Desktop is a work in progress.
+</InfoBlock>
+
+Transforms allow you to bind data with code in Qri, providing a quick means to automate the creation of new dataset versions from websites, APIs, and _other qri datasets_.
+
+To get started with transforms, check out [Transforms Qruickstart](https://qri.io/docs/transforms/quickstart).
+
+Below are several examples of transform scripts that demonstrate various approaches to automating qri datasets.  If you have a transform script that you think might help others, consider [opening a pull request](https://github.com/qri-io/website/pulls) to add it here.
+
+-- --
```

Comment:
I'd love to flip the order of these from less-to-more complex. What about:
* MTA Elevator Changes
* 311 Complaints By Boro
* World Bank Population

MTA should work in isolation, 311 involves geospatial calculations & a _big_ API fetch, and the last one requires you already have the `b5/country_codes` dataset before running

Reply:
Great idea, done